@uportal/esco-content-menu/src/components/ActionFavorites.vue

```diff
@@ -5,8 +5,8 @@
       backGroundIsDark ? 'background-dark' : '',
     ]"
     @click="toggleFavorite($event)">
-    <button class="favorite-button">
-      <span class="sr-only">{{ favoriteMessage }}</span>
```

Comment:
1. Will this bring back the issue where the screen reader is not reading off the `title` attribute?
2. Is the `sr-only` style available in the shadow DOM? I thought that was provided by Bootstrap?

Reply:
>  1. Will this bring back the issue where the screen reader is not reading off the `title` attribute?

Is the issue only with this component? If it is, could it be due to the fact that originally it was on a `div` element? Technically, it's valid HTML, but I never seen the `title` attribute on a `div` until now.

> 2. Is the `sr-only` style available in the shadow DOM? I thought that was provided by Bootstrap?

I believe it's available in the shadow DOM. grep-ing through the compiled file, I see the styles in there. Since this is reverting that change, then it's not a concern anymore?

Reply:
`sr-only` is provided only from frameworks, to avoid problems the best would be to define it locally with a simple css like:
`     .sr-only {
            position: absolute;
            width: 1px;
            height: 1px;
            padding: 0;
            margin: -1px;
            overflow: hidden;
            clip: rect(0, 0, 0, 0);
            white-space: nowrap; /* added line */
            border: 0;
      }`

After setting the favorite action need to use a button, it's an accessibility requirement.


About the title, it's an universal attribute that can be used nearly on all html tag : https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title It permits to have a tooltip.